ui/appui-react/src/appui-react/statusfields/MessageCenterField.tsx

```diff
@@ -0,0 +1,165 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Notification
+ */
+import * as React from "react";
+import {
+  Button,
+  NotificationMarker,
+  Popover,
+  Tabs,
+} from "@itwin/itwinui-react";
+
+import { SvgChat } from "@itwin/itwinui-icons-react";
+import type { MessageType } from "@itwin/core-react";
+import { Icon } from "@itwin/core-react";
+import { UiFramework } from "../UiFramework";
+import { OutputMessagePriority } from "@itwin/core-frontend";
+import { MessageCenterMessage } from "../layout/footer/message-center/Message";
+import { MessageManager } from "../messages/MessageManager";
+import { TitleBar } from "../layout/footer/dialog/TitleBar";
+import type { NotifyMessageDetailsType } from "../messages/ReactNotifyMessageDetails";
+import "../layout/footer/message-center/Dialog.scss";
```

Comment:
Imported scss file name should match tsx file name (should only contain CSS for that component and not be imported by other components).